### ISSUE TITLE
uncaught_exception does not work (in programs?)

### ISSUE THREAD

**Issue by sbcgua** [abaplint/abaplint#1613]
This is not catched (but SCI does)
```abap
  method lif_xxx_model~get_edoc_type. " Method of a local class in an include
    rv_edoc_type = zcl_ede_xxx_utils=>doc_type_to_edoc_type( ms_sel-doc_type ). " Raises
  endmethod.
```
Seems like it is not related to interface methods only, normal local method didn't work either.

**Comment by larshp** [comment]
any public code where it can be reproduced?

**Comment by sbcgua** [comment]
Nope, but give a couple of min ...

**Comment by sbcgua** [comment]
https://github.com/sbcgua/abaplint-bugz/blob/main/src/zabaplint_bugz_model.prog.abap#L13

**Comment by sbcgua** [comment]
25 min to recrate ... I was optimistic :)
